scripts/format.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Build a list of all the top-level directories in the project.
+for DIRECTORY in */ ; do
+  GOGLOB="$GOGLOB ${DIRECTORY%/}"
+done
+GOGLOB="${GOGLOB/ docs/}"
+GOGLOB="${GOGLOB/ vendor/}"
+
+GOFMT_LINES=`gofmt -s -l $GOGLOB | tr '\\\\' '/' | wc -l | xargs`
+
+if [[ $GOFMT_LINES -ne 0 ]]; then
+    FMT_FILES=`gofmt -s -l $GOGLOB | tr '\\\\' '/' | xargs`
+    for FILE in $FMT_FILES; do
+        echo "Running: gofmt -s -w $FILE"
+        `gofmt -s -w $FILE`
+    done
+fi
```

Comment:
How about adding above code in [makefile](https://github.com/prebid/prebidserver/blob/459fe58fb909e21baaed518078db957b7368e583/Makefile#L1)? With this change,

```
.phoney: format
 // code....
```

- User can run `make format` command to run `gofmt`
- Also we can update `validate.sh` to run `make format` command. This will eliminate code duplication between format.sh and validate.sh files

Reply:
This is good point! I'll make a change

Reply:
@onkarvhanumante please see these changes https://github.com/prebid/prebid-server/pull/2946/commits/ff28932321ca9f635efd51ffcce1e230445fedcf
I did not put a bash script itself to the Makefile, instead it calls this script from the sh file. I think it looks cleaner and aligns with overall style of the Makefile and validate.sh script. What do you think?